server/checks/PullRequestLabels.js

```diff
@@ -10,7 +10,7 @@ const error = logger(CHECK_TYPE, 'error')
 const debug = logger(CHECK_TYPE)
 const createStatePayload = getPayloadFn(CONTEXT)
 
-export function generateStatus(labels, checkConfig) {
```

Comment:
why the need for changing name here?

Reply:
to distinguish it clearly what is checked ... because there are now two functions generating a status here in this file